packages/grammar-selector/spec/grammar-selector-spec.js

```diff
@@ -287,10 +299,10 @@ describe('GrammarSelector', () => {
           }
         }
 
-        expect(cppCount).toBe(2); // ensure we actually saw two grammars
+        expect(cppCount).toBe(3); // ensure we actually saw two grammars
```

Comment:
```suggestion
        expect(cppCount).toBe(3); // ensure we actually saw three grammars
```

Small I know, but nice to keep things consistent